src/Config.php

```diff
@@ -105,7 +105,59 @@ protected static function defined($key)
             $message = "Aborted trying to redefine constant '$key'. `define('$key', ...)` has already been occurred elsewhere.";
             throw new ConstantAlreadyDefinedException($message);
         }
-        
+
         return false;
     }
+
+    /**
+     * Define set of config items.
+     *
+     * @param array definitions
+     * @return void
+     */
+    protected static function defineSet($definitions)
+    {
+        foreach ($definitions as $const => $def) {
+            self::define($const, $def);
+        }
+    }
+
+    /**
+     * Checks a value in the config map.
+     *
+     * @param string config key
+     * @return bool
+     */
+    protected static function is($key, $definition)
+    {
+        return self::$configMap[$key] == $definition;
+    }
```

Comment:
```suggestion
    /**
     * Asserts a value in the config map.
     *
     * @param string config key
     * @param mixed expected value
     * @return bool
     */
    protected static function assertEquals($key, $expected)
    {
        return self::$configMap[$key] == $expected;
    }
```

I'm not a fan of this method, doesn't seem necessary. But if we included it I would expect it to look like. Leaning towards leaving it out. Need a second opinion.

Reply:
I'd definitely leave this one out I suppose. Let people choose on their own whether to use `==` or `===`